packages/apps/dev-wallet/src/modules/account/account.service.ts

```diff
@@ -235,18 +243,38 @@ export async function fundAccount({
   address,
   keyset,
   profileId,
-  networkId = 'testnet04',
+  network,
```

Comment:
Do we need the full network here or is `networkUUID` sufficient?

Reply:
I need both uuid and networkId there. 
I can read networkId from db inside the function again but since the network object was already in the context to passed it down